src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let total;
+  const rentPrice = 40;
+
+  if (days >= 7) {
+    total = days * rentPrice - 50;
+  } else if (days >= 3 && days <= 6) {
+    total = days * rentPrice - 20;
+  } else {
+    total = days * rentPrice;
```

Comment:
Maybe count total outside the statement 'if'. To remove the unnecessary 'else'.

Reply:
Yeah, all of it make sense, I'll fix these issues. Thank you for remarks!